content/docs/iac/adopting-pulumi/migrating-to-pulumi/from-terraform.md

```diff
@@ -498,3 +499,178 @@ Before running the deployment the import file generated will be written out to t
 ### Example Conversion
 
 For an example of a full conversion, see the [Converting Full Terraform Programs to Pulumi](/blog/converting-full-terraform-programs-to-pulumi/) blog post.
+
+## Using Terraform Modules Directly
+
+Pulumi allows you to use existing Terraform modules directly in your Pulumi programs without converting or rewriting them. This feature is particularly useful for:
```

Comment:
I wonder if this is worth bringing up or it is just a pedantic curiosity - a plain file with `tf` declarations is technically a module with no inputs and outputs. So any existing Terraform code even if it does not use modules apriori can be integrated into Pulumi by quickly turning it into a module by adding varables.tf and outputs.tf.